samples/bookinfo/platform/consul/README.md

```diff
@@ -1,3 +1,7 @@
+# Disclaimer
+
+Warning: this example is not included in Istio's release verification testing, so it may not function exactly as documented.
```

Comment:
```suggestion
This example is not included in Istio's release verification testing, so it may not function exactly as documented.
```

Reply:
Actually, given the lint error, maybe it would be better to remove the disclaimer section heading and make this warning the first sentence of the `Consul Adapter for Istio on Docker` section. Start the sentence with `WARNING:` or `DISCLAIMER:`, maybe even in bold.